app/src/main/java/io/github/droidkaigi/confsched/viewmodel/event/EventBus.java

```diff
@@ -0,0 +1,24 @@
+package io.github.droidkaigi.confsched.viewmodel.event;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+import rx.subjects.SerializedSubject;
+import rx.subjects.Subject;
+
+public class EventBus {
```

Comment:
Quick question, Is it common way to use EventBus to notify View from ViewModel? Sorry for my ignorance... 😇